doc/nrf/ug_nrf9160_gs.rst

```diff
@@ -0,0 +1,502 @@
+
+   .. figure:: /images/programmer_write_nrf9160dk.png
+      :alt: Programmer - Write
+
+      Programmer - Write
+
+   The :guilabel:`Modem DFU` dialog window appears.
+
+   .. figure:: images/programmerapp_modemdfu.png
+      :alt: Modem DFU window
+
+      The Modem DFU window
+
+#. Click the :guilabel:`Write` button in the :guilabel:`Modem DFU` dialog window to update the firmware.
+   Do not unplug or turn off the device during this process.
+
+When the update is complete, you see a success message.
+
+If you experience any problems during the process, restart the Programmer app by pressing Ctrl+R (Cmd+R on macOS), and try again.
+
+.. note::
+
+   If you experience issues updating the modem firmware, click :guilabel:`Erase all` in the :guilabel:`Device` section of the side panel and try updating again.
+
+   Doing this erases the contents of the flash memory, so the application must programmed again to the DK.
+
+.. _nrf9160_gs_updating_cloud_certificate:
+
+Updating the nRF Cloud certificate
+**********************************
+
+To test the nRF9160 DK with the :ref:`asset_tracker_v2` application, you need an `nRF Cloud`_ account.
+nRF Cloud is Nordic Semiconductor's platform for connecting your IoT devices to the cloud, viewing and analyzing device message data, prototyping ideas that use Nordic Semiconductor's chips, and more.
+It also includes a public REST Application Programming Interface (API) that can be used for building your own IoT solutions.
+
+Since the application transmits data to nRF Cloud for visualization, it requires a valid security certificate.
+This means that you might need to update the certificate stored on the DK.
+To do so, you need to download a new certificate and then provision it to your DK.
+
+.. _creating_cloud_account:
+
+Creating an nRF Cloud account
+=============================
+
+To create an nRF Cloud account, complete the following steps:
+
+1. Go to `nRF Cloud`_ and click :guilabel:`Register`.
+#. Enter your email address and choose a password, then click :guilabel:`Create Account`.
+   nRF Cloud will send you a verification email.
+#. Copy the 6-digit verification code and paste it into the registration dialog box.
+   If you do not see the verification email, check your junk mail for an email from no-reply@verificationemail.com.
+
+   If you closed the registration dialog box, you can repeat Step 1 and then click on :guilabel:`Already have a code?`.
+   Then enter your email and the verification code.
+
+You can now sign in on `nRF Cloud`_ with your email and the password your chose.
+After signing in, you are taken to the dashboard view that displays your device count and service usage.
+
+.. _downloading_cloud_certificate:
+
+Downloading the nRF Cloud certificate
+=====================================
+
+Complete the following steps to download the nRF Cloud certificate for your nRF9160 DK:
+
+1. Go to `nRF Cloud`_ and sign in.
```

Comment:
"Sing in to nRF Cloud"?

Reply:
I kept it this way so it aligns with "Go to nRF Cloud and click Register" from the previous section.
Can be changed, though.